### ISSUE TITLE
Openstack Cloud Provider shouldn't require user_id

### ISSUE THREAD

**Issue by davidarcher** [rancher/terraform-provider-rancher2#18]
When defining a cluster that uses Rancher-launched Kubernetes on OpenStack, I have to put a blank value for `user_id`. The actual requirement is to have either `username` or `user_id` specified but it seems we're requiring BOTH fields to be defined here.

```
resource "rancher2_cluster" "cluster" {
  name        = "${var.cluster_name}"
  description = "${var.cluster_description}"

  rke_config {
    cloud_provider {
      openstack_cloud_provider {
        global {
          auth_url    = "${var.openstack_auth_url}"
          tenant_id   = "${var.openstack_tenant_id}"
          domain_name = "${var.openstack_domain_name}"
          user_id     = ""    ### I shouldn't have to do this
          username    = "${var.openstack_username}"
          password    = "${var.openstack_password}"
        }
...
```

**Comment by rawmind0** [comment]
Removed `used_id` field on global argument in favour of `username` following [k8s openstack cloud provider docs](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/provider-configuration.md#global-required-parameters)

Fixed on PR #8

**Comment by rawmind0** [comment]
Fix for this issue is included on new release [v1.1.0](https://github.com/terraform-providers/terraform-provider-rancher2/releases/tag/v1.1.0) 

@davidarcher, please reopen issue if needed.
